### DISCUSSION TITLE
Details about tomorrow interactive session

### DISCUSSION THREAD

**Discussion by Daipayan-Das** [github/india#323] [Students]
Hey, mentors can you please provide me with the details of the interactive session that was about to be held on 8th October?....Is there any meeting link to join the program?

**Comment by Daipayan-Das** [comment]
will i get any answer???

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
